### DISCUSSION TITLE
Call to a member function with() on null

### DISCUSSION THREAD

**Discussion by Instartec** [livewire/livewire#6656] [7. Bugs (archived)] [answered]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a function with $this->redirect('/any-route')->with('hello', 'bye');

### Additional information

Hi guys, i did try put with function in my function, but it doesn't works, it returns to me "Call to a member function with() on null".
![image](https://github.com/livewire/livewire/assets/144027080/d23365c1-c0fa-4dfe-be4b-d314de7ad56f)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ali-hasnain-dev** [comment]
Write code in mount function

public function mount()
    {
       return $this->redirect('/any-route')->with('hello', 'bye');
    }

**Reply by Instartec** [reply]
Them remove this from the documentation...was a bad bug.

**Comment by danie-ramdhani** [comment, marked as answer]
the chained function `with` is not exist in livewire. https://github.com/livewire/livewire/discussions/8128#discussioncomment-8808143.
the documentation is fixed in https://github.com/livewire/livewire/pull/8131
